addon.xml

```diff
@@ -4,6 +4,7 @@
     <import addon="script.module.requests" version="2.12.4" />
     <import addon="script.common.plugin.cache" version="2.5.5" />
     <import addon="script.module.urlresolver" version="5.0.14" />
+    <import addon="script.module.beautifulsoup4" version="4.5.3"/>  
```

Comment:
`script.module.beautifulsoup4`
is this served by default kodi?
or we will need to import it to our repo?
also, i havnt seen it was used, if this is a leftover of some test, lets remove it